### PR TITLE
Update braze-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/braze-cohort.md [hunk 1]
```diff
@@ -64,3 +64,10 @@ To sync your first cohort, follow these steps:
 2. Click on **Create Segment** and name your Segment.
 3. Under Add Filter, select **Amplitude Cohorts** and choose the cohort you created and synced from Amplitude.
 4. When saved, you can reference this segment during Canvas or campaign creation in the targeting users step.
+
+### Frequently asked questions
+
+??? note "Why can't I see the list of users in Braze?"
```
Comment (markzegarelli) [inline docs/data/destinations/braze-cohort.md:70]:
```suggestion
### Why can't I see the list of users in Braze?
```

File: docs/data/destinations/braze-cohort.md [hunk 2]
```diff
@@ -64,3 +64,10 @@ To sync your first cohort, follow these steps:
 2. Click on **Create Segment** and name your Segment.
 3. Under Add Filter, select **Amplitude Cohorts** and choose the cohort you created and synced from Amplitude.
 4. When saved, you can reference this segment during Canvas or campaign creation in the targeting users step.
+
+### Frequently asked questions
```
Comment (markzegarelli) [inline docs/data/destinations/braze-cohort.md:68]:
```suggestion
## Frequently asked questions
```

File: docs/data/destinations/braze-cohort.md [hunk 3]
```diff
@@ -64,3 +64,10 @@ To sync your first cohort, follow these steps:
...
+
+??? note "Why can't I see the list of users in Braze?"
+
+    One common issue is that the users in the Amplitude cohort may not exist in Braze. When you sync to Braze, new users will not be created if they do not already exist in Braze. Therefore, it's important to ensure that all the users in your Amplitude cohort have a matching User ID value in Braze.
```
Comment (markzegarelli) [inline docs/data/destinations/braze-cohort.md:72]:
```suggestion
When you sync a cohort, Braze matches users in that cohort with users that already exist in Braze, and doesn't create new users. Make sure users in your Amplitude cohort have a matching user (`userId`) in Braze.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/data/destinations/braze-cohort.md [hunk 4]
```diff
@@ -64,3 +64,10 @@ To sync your first cohort, follow these steps:
...
+
```
Comment (markzegarelli) [inline docs/data/destinations/braze-cohort.md:73]:
```suggestion
```

**Comment by Brandoncharleskhoo1** [comment]
@markzegarelli Hi Mark, i'm not sure what else I have to do here. I already committed your suggestions :)

**Comment by markzegarelli** [comment]
Moved to new site

**Pull request closed without merging**